docs/server-instructions.md

```diff
@@ -0,0 +1,102 @@
+# Add the JVM agent to Java servers
+
+The following sections show how to add the path to the Splunk OTel agent for Java using the [supported servers](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#application-servers).
+
+## JBoss EAP / WildFly
```

Comment:
Can we link to the official docs for configuring these servers with a javaagent or JVM parameter, as well?

Reply:
Most servers lack simple docs on adding JVM parameters, and they tend to change between versions. Even when they do have docs, they present instructions that are either too generic or too difficult to follow. With this list, we're actually providing some added value.